.github/workflows/pr-auto-fix.yaml

```diff
@@ -5,13 +5,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Cache bazel build
+      - name: Get current time
+        uses: srfrnk/current-time@master
+        id: current-time
+        with:
+          format: YYYYWW
       - name: Cache bazel
         uses: actions/cache@v2
         env:
           cache-name: bazel-cache
         with:
           path: ~/.cache/bazel
-          key: ${{ runner.os }}-${{ env.cache-name }}
+          key: ${{ runner.os }}-${{ steps.current-time.outputs.formattedTime }}
```

Comment:
It took me a second to figure out how this worked. Maybe a comment explaining what's going on here?